cnd/src/network.rs

```diff
@@ -1188,90 +1197,82 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<comit_ln::BehaviourOutEvent> fo
-
-                        tokio::task::spawn({
-                            // This is Bob
-                            let connector = self.ethereum_connector.clone();
-                            let alice_ethereum_identity = ethereum_identity;
-                            let bob_ethereum_identity = create_swap_params.ethereum_identity;
-
-                            let asset = create_swap_params.ethereum_amount.clone();
-                            let ledger = ledger::Ethereum::default();
-                            let expiry = create_swap_params.ethereum_absolute_expiry;
-                            let secret_hash = secret_hash;
-
-                            self::han::new_han_ethereum_ether_swap(
-                                local_swap_id,
-                                connector,
-                                self.alpha_ledger_states.clone(),
-                                HtlcParams {
-                                    asset,
-                                    ledger,
-                                    redeem_identity: bob_ethereum_identity.into(),
-                                    refund_identity: alice_ethereum_identity,
-                                    expiry,
-                                    secret_hash,
-                                },
-                                role,
-                            )
-                        });
+                match self.lnd_connector_params {
+                    None => { tracing::error!("Internal Failure: lnd connectors are not initialised, no action has been taken. This should be unreachable.") }
+                    Some(ref lnd_connector_params) => {
+                        match role {
+                            Role::Alice => {
+                                tokio::task::spawn({
+                                    let lnd_connector: LndConnectorAsReceiver = (**lnd_connector_params).clone().into();
+                                    self::halight::new_halight_swap(local_swap_id, secret_hash, self.halight_states.clone(), lnd_connector)
+                                        .instrument(
+                                            tracing::error_span!("beta_ledger", swap_id = %local_swap_id, role = %role),
+                                        )
+                                });
+
+                                tokio::task::spawn({
+                                    let connector = self.ethereum_connector.clone();
+                                    let alice_ethereum_identity = create_swap_params.ethereum_identity;
+                                    let bob_ethereum_identity = ethereum_identity;
+
+                                    let asset = create_swap_params.ethereum_amount.clone();
+                                    let ledger = ledger::Ethereum::default();
+                                    let expiry = create_swap_params.ethereum_absolute_expiry;
+                                    let secret_hash = secret_hash;
```

Comment:
Cut'n'pasta / sed residue?

Reply:
This was here before :)

Reply:
Oh, we should definitely remove it then, its probably just a relic of hacking fast during LN squad.